client/src/guestsAndCalc_mod3/Dropdown.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import DropdownContents from './DropdownContents';
+
+export const Holder = styled.div`
```

Comment:
I might suggest consolidating your styled components into separate files and importing them into your component to reduce the clutter inside of your component file. Also, is there a reason you are exporting all of your styled components?

Reply:
for my tests lol

Reply:
I'm thinking about doing this, but after everything is done.